### DISCUSSION TITLE
May I ask, is the issue I mentioned a bug? How can it be resolved?

### DISCUSSION THREAD

**Discussion by liyanshuai2018** [langgenius/dify#3656] [General]
1、Why is the response from calling large models within Ollama through the command line very quick, while it is much slower through Dify, even by a factor of ten to twenty times slower? Additionally, calling Ollama through open-webui is also more than ten times faster than through Dify. What is the reason for this discrepancy?
2、When uploading an image and calling the LLaVA-13B model of Ollama via Dify, the following prompt appears: "Query or prefix prompt is too long, you can reduce the prefix prompt, or shrink the max token, or switch to an LLM with a larger token limit size." However, when calling Ollama through the local command line or open-webui, this issue does not occur. Is there a need to adjust any parameters?

**Comment by crazywoola** [comment]
> Query or prefix prompt is too long, you can reduce the prefix prompt, or shrink the max token, or switch to an LLM with a larger token limit size.

In conversation orchestration page, you need to adjust the max token of chosen ollama model. This meant to be the length of output token. It's different with the whole input tokens.

Regarding the first question, please read this thread. https://github.com/ollama/ollama/issues/2851

**Comment by liyanshuai2018** [comment]
> > Query or prefix prompt is too long, you can reduce the prefix prompt, or shrink the max token, or switch to an LLM with a larger token limit size.
> 
> In conversation orchestration page, you need to adjust the max token of chosen ollama model. This meant to be the length of output token. It's different with the whole input tokens.
> 
> Regarding the first question, please read this thread. [ollama/ollama#2851](https://github.com/ollama/ollama/issues/2851)
@crazywoola v0.57 works fine, but v0.64 can't add models and gives a 502 error
![image](https://github.com/langgenius/dify/assets/42138577/08c05a0c-d2c1-4eac-9363-02ba8c1cfe1a)

**Reply by liqi0706401043** [reply]
Do you hava solved?

**Comment by liqi0706401043** [comment]
Do you hava solved this problem? I'm also meet the same situation.And when I according to the document to adjust the model max token, it  not work.God help me !
